src/styles/blocks/main.scss

```diff
@@ -0,0 +1,26 @@
+.container {
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
```

Comment:
Similar to the previous comment, using 'minmax(200px, 1fr)' for grid-template-columns will provide a more fluid and responsive layout.